x-pack/plugins/integration_assistant/server/integration_builder/data_stream.ts

```diff
@@ -31,6 +32,8 @@ export function createDataStream(
       data_stream_description: description,
       package_name: packageName,
       data_stream_name: dataStreamName,
+      multiline_json_toggle_display: logFormat.startsWith('ndjson'),
```

Comment:
Let's set this one to only `ndjson+multiline`, since the nornal ndjson is always on a single line, that way we don't get the overhead of multiline when its not needed.

Reply:
The idea here is that the user might know that _sometimes_ there are multi lines and can set the toggle (provided they are aware of "Advanced Options") manually. 

Though I did add a risk that some users will set the toggle even if not necessary 🤷